buildings/tests/run.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import traceback
+import signal
+import importlib
+import unittest
+from pathlib import Path
+
+from qgis.core import QgsApplication
+from qgis.PyQt.QtCore import QDir
+from qgis.utils import iface
+
+from console.console_output import writeOut
+
+
+if iface is None:
```

Comment:
perhaps include this in a try-except of the import of iface so is caught cleanly and returned with the error